src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown

```diff
@@ -461,6 +462,23 @@ The lowest nightly rate averaged over the stay.
 `avgNightlyRate`|[`Price`](#schemaprice)|-|**Required** |
 `isTaxAndFeesInclusive`|`boolean`|`true` / `false`|Whether or not tax and fees are included.|
 
+## <a id="schematotalpricehotelproperty"></a> TotalPrice
+
+Details about the total pricing associated with stay.
+
+|Name|Type|Format|Description|
+|---|---|---|---|
+`totalOverStay`|[`TotalOverStay`](#schematotalOverStay)|-|**Required** |
+`isTaxInclusive`|`boolean`|`true` / `false`|Whether or not taxes are included.|
+`isFeesInclusive`|`boolean`|`true` / `false`|Whether or not fees are included.|
+
+## <a id="schematotalOverStay"></a> TotalOverStay
```

Comment:
we already have #schemaprice with this structure, just use that, no need for new type

Reply:
schematotalOverStay removed